zamba/models/config.py

```diff
@@ -696,6 +694,7 @@ def validate_files(cls, values):
             logger.warning(
                 f"Found {num_duplicates} duplicate row(s) in filepaths csv. Dropping duplicates so predictions will have one row per video."
             )
+            files_df = files_df[["filepath"]].drop_duplicates()
```

Comment:
Do we want this behavior at all? What about interspecies friendships? I think we want to allow duplicates that just get coalesced during OHE since our models are multiclass anyway.

Reply:
oops, sorry. just realized this is predict not train. nevermind me

Reply:
yep this just avoids having multiple rows per video written out in preds, as i experienced